### ISSUE TITLE
Switch block for loop tokens

### ISSUE THREAD

**Issue by nfour** [gandm/language-babel#179]
```js

    const fn = function (state = initialState, { type, payload }) {
        switch (type) {
            case `${key}__SET`: {
                state = { ...state }
                
                for ( let item in payload )
                    state.items = setItem( state.items, resolveRelations({ ...item }) )

                return state
            }
            
            case `${key}__DELETE`:
                return { ...state, items: deleteItem(state.items, payload) }

            default: {
                return state
            }
        }
    }

```
In this example the `for` in the switch case for `SET` is tokenized as `entity name function method js` instead of `keyword control loop js`.

Removing the braces around the case fixes it, but I have found in some cases it's illegal to use `let` and `const` inside a case block without the braces.

**Comment by gandm** [comment]
Thanks for the report - fixed in 2.20.3. I did a large refactor on 2.20.0 trying to distinguish methods better. e.g. using keywords as methods inside classes.
```js
class A {
  // These will be converted to methods
  if (a) {}; for (a) {}; switch (a) {}
}


function a(state ) {
    // These will be converted to conditional/loops...
    if (a) {}; for (a;;) {}; switch (a) {}
    switch (type) {
         case a: {
           for ( let item in payload )
                state.items = setItem( state.items, resolveRelations({ ...item }) )
            return state
        }
    }
}
```

**Comment by nfour** [comment]
Great work overall @gandm, I'm really impressed with this library.
